interfaces/inc/file.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef FILE_H
+#define FILE_H
+
+#ifdef __cplusplus
+#include <cstdint>
+#else
+#include <stdbool.h>
+#include <stdint.h>
+#endif
+
+#include "umock_c/umock_c_prod.h"

```

Comment:
needs to stay as close to MOCKABLE_FUNCTION as possible.

Reply:
I moved this right above the first MOCKABLE FUNCTION. Is that okay?

---
In reply to: [450465330](https://github.com/Azure/azure-c-pal/pull/7#discussion_r450465330) [](ancestors = 450465330)

Reply:
I think he means like above line 18. Nothing should be included within the extern C block.

---
In reply to: [450532400](https://github.com/Azure/azure-c-pal/pull/7#discussion_r450532400) [](ancestors = 450532400,450465330)

Reply:
I moved it to be the last include.

---
In reply to: [451685834](https://github.com/Azure/azure-c-pal/pull/7#discussion_r451685834) [](ancestors = 451685834,450532400,450465330)